list-cookies/cookies.js

```diff
@@ -0,0 +1,13 @@
+window.onload = function(e){
+    //get all cookies in the domain
```

Comment:
The convention usually followed in this repo is for 2-space indentation. Sorry, we should mention this :-/